src/errors.ts

```diff
@@ -63,6 +86,7 @@ export enum GeneralErrorCodes {
     OrderSubmissionDisabled = 102,
     Throttled = 103,
     NotImplemented = 104,
+    TransactionInvalid = 105,
 }
```

Comment:
We may want to re-think these conventions but ok to move forward with them for now.

Reply:
Agreed, this doesn't leave much room for SRA related things, though SRA being essentially deprecated makes this moot.

As far as I can tell (from our consumer libraries) these codes aren't used directly. It's consistent, but I'm not sure it adds value.